### PR TITLE
Remove things in __init__.py in app_opt

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Remove things in `__init__.py` in app_opt/he and app_opt/pt
- Fix imports for other files

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_common/pt/pt_fed_utils.py [hunk 1]
```diff
@@ -18,5 +18,7 @@
     f"This module: {__file__} is deprecated. Please use nvflare.app_opt.pt.", category=FutureWarning, stacklevel=2
```
Comment (holgerroth) [inline nvflare/app_common/pt/pt_fed_utils.py:18]: Should the warning say `nvflare.app_opt.pt.model_persistence_format_manager`?

Comment (YuanTingHsieh) [reply nvflare/app_common/pt/pt_fed_utils.py:18]: previously this `model_persistence_format_manager` is put inside `pt_fed_utils.py`
but right now in app_opt I separate them, what do you think should be the correct warning?

Comment (holgerroth) [reply nvflare/app_common/pt/pt_fed_utils.py:18]: I see. Please ignore my comment. The current warning should be fine then.

**Pull request merged**